test-integration/services/blockchain/contracts/IDContract.test.ts

```diff
@@ -36,7 +36,7 @@ describe('IDContract class', () => {
 
         // Close connections
         await user.logout();
-    });
+    }, 8000);
```

Comment:
what is this?

Reply:
it throws 5000 timeout error so i added more